grin.toml

```diff
@@ -33,6 +33,9 @@ db_root = ".grin"
 #
 #seeds = ["192.168.0.1:13414","192.168.0.2:13414"]
 
+#A list of preferred peers to connect to
```

Comment:
Can we nest this under p2p config so it sits alongside `peers_allow` and `peers_deny`?